docs/development/virtual_environments.rst

```diff
@@ -100,22 +40,28 @@ course, you can specify the name if you want to.
    pytest -v tests/        # run the unit and functional tests
 
 SecureDrop consists of two separate web applications (the Source Interface and
-the Journalist Interface) that run concurrently. The development servers will
-detect code changes when they are saved and automatically reload.
+the Journalist Interface) that run concurrently. In the Development environment
+they are configured to detect code changes and automatically reload whenever a
+file is saved. They are made available on your host machine by forwarding the
+following ports:
+
+* Source Interface: ``localhost:8080``
+* Journalist Interface: ``localhost::8081``
```

Comment:
I'm going to make these URLs, so a develop running the `development` VM can just click on them to pull up the relevant interfaces.